package.json

```diff
@@ -29,10 +29,7 @@
     "@babel/eslint-parser": "7.21.8",
     "@changesets/cli": "2.24.1",
     "@types/jest": "26.0.23",
-    "@types/react": "17.0.4",
-    "@types/react-native": "0.64.4",
-    "@types/styled-components": "5.1.25",
-    "@types/styled-components-react-native": "5.1.3",
+    "@types/react": "18.2.24",
```

Comment:
styled-components, react-native types not needed?

Reply:
Oh just saw they are added in blade's package.json. So why is types/react needed here but types/styled-components are in blade's package.json?